packages/surveys/README.md

```diff
@@ -31,9 +31,18 @@ const exitSurvey = { type: chsSurvey.ExitSurveyPlugin };
 
 ### Parameters
 
```

Comment:
Is it worth adding the same "#### Optional" subheading here under "Parameters", for clarity/consistency? And maybe the same for the video config plugin (around line 22)?